packages/web/app/src/pages/target.tsx

```diff
@@ -209,41 +164,23 @@ function SchemaView(props: {
   organization: FragmentType<typeof SchemaView_OrganizationFragment>;
   project: FragmentType<typeof SchemaView_ProjectFragment>;
   target: FragmentType<typeof SchemaView_TargetFragment>;
-  highlightedService: string | null;
 }): ReactElement | null {
   const organization = useFragment(SchemaView_OrganizationFragment, props.organization);
   const project = useFragment(SchemaView_ProjectFragment, props.project);
   const target = useFragment(SchemaView_TargetFragment, props.target);
-  const [filterService, setFilterService] = useState(props.highlightedService ?? '');
-  const [term, setTerm] = useState(props.highlightedService ?? '');
-  const [openItems, setOpenItems] = useState<string[]>([]);
```

Comment:
There's no point of keeping open items in a state. The accordion does it for us.
The `filterService` is not really used.
The `term` is also not needed as combobox is handling the search input for us.